homeassistant/components/sensor/bom.py

```diff
@@ -188,6 +189,7 @@ def __init__(self, hass, station_id):
         self._hass = hass
         self._zone_id, self._wmo_id = station_id.split('.')
         self._data = None
+        self._lastupdate = LAST_UPDATE
```

Comment:
What do you think about having `self._last_updated` instead? I think it might be a slightly more logical variable name.